uk.co.compucorp.civicrm.hrleaveandabsences/js/test/leave-type-wizard/components/leave-type-wizard.component.spec.js

```diff
@@ -207,6 +207,28 @@ define([
             expect(controller.sections[0].active).toBe(true);
           });
         });
+
+        describe('when user clicks the "next" button 2 times and then "back" 1 time', function () {
+          beforeEach(function () {
+            controller.goNext();
+            controller.goNext();
+            controller.goBack();
+          });
+
+          it('opens seconds tab', function () {
```

Comment:
Seems there is a typo here. Guess you meant `opens second tab` and not `seconds`.

Reply:
Fixed.